### ISSUE TITLE
Exception in callback of async function: _initIOs

### ISSUE THREAD

**Issue by rsylvian** [lpender/meteor-pushwoosh#7]
Hey!

I'm using this plugin since two weeks and it's working great! Thanks for making it 👍 

I'm having one small issue tho...
First time you open the app it works like a charm. If you close or kill the app and relaunch :

```
Exception in callback of async function: _initIOs@http://meteor.local/24f8b967c8bb1cfb8af73fd7be34b646a302b2f5.js?meteor_js_resource=true:223:732
initPushwoosh@
....
```

Not sure if it's related but I also get a cordova warning in the same time:

```
Snapshotting a view that has not been rendered results in an empty snapshot. 
Ensure your view has been rendered at least once before snapshotting or snapshot after screen updates.
```

It causes my app to hang indefinitely since the `next()` in the router is never called.

Any idea?

**Comment by lpender** [comment]
Thanks for the bug report.

Hmm. I don't have any ideas but I did just go through and refactor/clean up the code a bit. 

Mind updating to the latest version and trying again? 

Thanks
-L

**Comment by lpender** [comment]
This is certainly the function that is throwing an exception:

https://github.com/lpender/meteor-pushwoosh/blob/master/cordova.js#L35

Might want to fork the repo locally and do some logging around it to see if you can determine exactly where it's breaking.

I haven't tried to reproduce it in depth, but it seems to be working for me.

**Comment by rsylvian** [comment]
@lpender Awesome thanks! I'm gonna give it a try tomorrow morning.

**Comment by rsylvian** [comment]
```
module com.pushwoosh.plugins.pushwoosh.PushNotification not found
```

According to the pushwoosh cordova plugin documentation it looks like the right way to include is:

``` js
cordova.require("pushwoosh-cordova-plugin.PushNotification");
```

Issue related: https://github.com/phonegap/phonegap-app-developer/issues/371

**Comment by lpender** [comment]
That's interesting--I wonder why it's working at all?

I can make that change and do some testing, thx

**Comment by lpender** [comment]
Hey, published a new version, 2.1.7, with that change. 

Let me know if that works

**Comment by lpender** [comment]
@MyBoon any luck?
